src/gtk/toga_gtk/window.py

```diff
@@ -117,6 +117,7 @@ def set_position(self, position):
         pass
 
     def set_size(self, size):
+        self.native.resize(size[0],size[1])
         pass
```

Comment:
The pass is redundant if there's body content.

Reply:
I should probably get into the habit of reading my own commits before they get reviewed. :sweat: Sorry